dspace-server-webapp/src/test/java/org/dspace/app/rest/DiscoveryRestControllerIT.java

```diff
@@ -1140,6 +1142,58 @@ public void discoverSearchObjectsTest() throws Exception {
         ;
     }
 
+    @Test
+    public void discoverSearchObjectsWithSpecialCharacterTest() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context).build();
+        Collection collection = CollectionBuilder.createCollection(context, parentCommunity).build();
+        ItemBuilder.createItem(context, collection)
+                .withAuthor("Atmire & friends")
```

Comment:
Tiny thing, but I'd recommend keeping test names entirely generic (as I don't want each service provider getting in the habit of inserting their name in our shared tests).  Could we change this to say "DSpace & friends" or give a generic person name like "Smith, Jane"?

Reply:
Hi Tim, I understand, I've changed it to 'DSpace & friends'.